tests/example-syntax-error/expected_results.json

```diff
@@ -5,7 +5,7 @@
   "tests": [
     {
       "name": "Bob #hey responds to stating something",
-      "test_code": "Bob.hey(\"Tom-ay-to, tom-aaaah-to.\").should eq \"Whatever.\"",
+      "test_code": "(Bob.hey(\"Tom-ay-to, tom-aaaah-to.\")).should(eq(\"Whatever.\"))",
```

Comment:
The parenthesis for `should` is not commonly used, and there seems be extra parenthesis before `.should` as well.

Reply:
Yea not ideal, but at this point trying to consistently fix a broken test runner. I agree that there is room for improvement here to match the exact formatting of the original source. 

I believe the AST Node base class has some source position data associated with it (which I will be looking at next). I believe this shouldn't block this fix though as it is still valid crystal syntax

Reply:
Yep, hence the comment rather than requested change.

Reply:
👍🏻 I think I got it figured out, so I pushed a commit with that solution added